src/index.js

```diff
@@ -353,7 +353,6 @@ class MiniCssExtractPlugin {
                   '}',
                   'var linkTag = document.createElement("link");',
                   'linkTag.rel = "stylesheet";',
-                  'linkTag.type = "text/css";',
```

Comment:
Why we need remove this? Not sure it is right idea

Reply:
Removing `text/css` good for HTML5, but some browsers (include mobile) still have bad supporting HTML5

Reply:
The default value of the `type` attribute has always been `text/css`. The only difference in HTML5 was making it pass validation when omitted. There is no functionality change with regard to this behaviour between the two specs.

You can test this yourself - load this webpage in IE6:

```html
<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN" "http://www.w3.org/TR/html4/strict.dtd">
<html lang="en">
  <head>
    <title>HTML 4 Strict</title>
    <link rel="stylesheet" href="style.css">
    <!-- make sure style.css exists and and is not empty -->
  </head>  
  <body>
    <script type="text/javascript">
      alert(document.styleSheets.length);
    </script>
  </body>
</html>
```

<img width="303" alt="alert 1 in IE6" src="https://user-images.githubusercontent.com/105127/50850225-3c7de480-1347-11e9-86b0-3a477f7daf0c.png">

Reply:
Nicely done